Providers/Modules/NPM/Plugin/conf/plugin.conf

```diff
@@ -0,0 +1 @@
+this is a 32-bit binary placeholder
```

Comment:
we dont need that right now, our ruby plugin runs agnostic to platform architecture